sentences/sr/_common.yaml

```diff
@@ -1,12 +1,57 @@
 language: sr
 responses:
   errors:
-    no_intent: TODO Sorry, I couldn't understand that
-    no_area: TODO No area named {{ area }}
-    no_domain: TODO {{ area }} does not contain a {{ domain }}
-    no_device_class: TODO {{ area }} does not contain a {{ device_class }}
-    no_entity: TODO No device or entity named {{ entity }}
-    handle_error: TODO An unexpected error occurred while handling the intent
-lists: {}
-expansion_rules: {}
-skip_words: []
+    no_intent: "Нажалост, нисам вас разумео"
+    no_area: "Не постоји просторија са именом {{ area }}"
+    no_domain: "Просторија {{ area }} не постоји у оквиру подручја {{ domain }}"
+    no_device_class: "Просторија {{ area }} не садржи {{ device_class }}"
+    no_entity: "Не постоји ни уређај ни ентитет под именом {{ entity }}"
+    handle_error: "Дошло је до неочекиване грешке током обраде вашег захтева"
+lists:
+  color:
+    values:
+      - in: "бел(а|o)"
+        out: "white"
+      - in: "црн(а|o)"
+        out: "black"
+      - in: "црвен(а|o)"
+        out: "red"
+      - in: "наранџаст(а|o)"
+        out: "orange"
+      - in: "жут(а|o)"
+        out: "yellow"
+      - in: "зелен(а|o)"
+        out: "green"
+      - in: "плав(а|o)"
+        out: "blue"
+      - in: "љубичаст(а|o) | лила"
+        out: "purple"
+      - in: "браон"
+        out: "brown"
+      - in: "ружичаст(а|o) | розе"
+        out: "pink"        
+  brightness:
+    range:
+      type: "проценат(а)"
+      from: 0
+      to: 100
+  temperature:
+    range:
+      type: "температура"
+      from: 0
+      to: 100
+  temperature_unit:
+    values:
+      - in: "Ц | [степен[а|и]] Целзијус[а]"
+        out: "celsius"
+      - in: "Ф | [степен[а|и]] Фаренхајт[а]"
+        out: "fahrenheit"
+expansion_rules:
+  area: "[у | просторији] {area}"
+  turn_on: "(упали[ш|те|мо] | укључи[ш|те|мо])"
+  turn_off: "(угаси[ш|те|мо] | искључи[ш|те|мо])"
```

Comment:
Translate expansion rule keys `turn_on` and `turn_off`, because they do not wrap slots.